examples/Workflow/WorkflowConsoleApp/Models.cs

```diff
@@ -6,4 +6,10 @@ public record InventoryResult(bool Success, InventoryItem orderPayload);
     public record PaymentRequest(string RequestId, string ItemName, int Amount, double Currency);
     public record OrderResult(bool Processed);
     public record InventoryItem(string Name, double PerItemCost, int Quantity);
+    public enum ApprovalResult
+    {
+        Unspecified = 0,
+        Approved,
+        Rejected,
```

Comment:
Nit, I prefer they all have the number in there.

Reply:
Totally fine by me. Done.